tests/unit/utils/test_openstack.py

```diff
@@ -76,3 +76,75 @@ def test_generate_lookup(service):
         assert openstack_lookup[service][os_release] == VersionRange(
             version + ".0.0", str(int(version) + 1) + ".0.0"
         )
+
+
+@pytest.mark.parametrize(
+    "release_1, release_2, comparison, expected_result",
+    [
+        ("victoria", "wallaby", "eq", False),
+        ("victoria", "victoria", "eq", True),
+        ("victoria", "wallaby", "neq", True),
+        ("wallaby", "wallaby", "neq", False),
+        ("victoria", "wallaby", "lt", True),
+        ("wallaby", "victoria", "lt", False),
+        ("victoria", "wallaby", "le", True),
+        ("wallaby", "victoria", "le", False),
+        ("wallaby", "wallaby", "le", True),
+        ("victoria", "wallaby", "ge", False),
+        ("wallaby", "victoria", "ge", True),
+        ("victoria", "wallaby", "gt", False),
+        ("wallaby", "victoria", "gt", True),
+    ],
+)
+def test_compare_openstack(release_1, release_2, comparison, expected_result):
+    if comparison == "eq":
+        result = OpenStackRelease(release_1) == release_2
+        result_alternative = OpenStackRelease(release_1) == OpenStackRelease(release_2)
+    elif comparison == "neq":
+        result = OpenStackRelease(release_1) != release_2
+        result_alternative = OpenStackRelease(release_1) != OpenStackRelease(release_2)
+    elif comparison == "lt":
+        result = OpenStackRelease(release_1) < release_2
+        result_alternative = OpenStackRelease(release_1) < OpenStackRelease(release_2)
+    elif comparison == "le":
+        result = OpenStackRelease(release_1) <= release_2
+        result_alternative = OpenStackRelease(release_1) <= OpenStackRelease(release_2)
+    elif comparison == "ge":
+        result = OpenStackRelease(release_1) >= release_2
+        result_alternative = OpenStackRelease(release_1) >= OpenStackRelease(release_2)
+    elif comparison == "gt":
+        result = OpenStackRelease(release_1) > release_2
+        result_alternative = OpenStackRelease(release_1) > OpenStackRelease(release_2)
+    assert result == expected_result
+    assert result_alternative == expected_result
```

Comment:
Please do not use something like this, where we define lot of if statements. It would be better to use multiple simple tests or if you really want you can use `eval`, or call directly `__eq__` function.